src/analyses/everMultiThreaded.ml

```diff
@@ -0,0 +1,47 @@
+(** Analysis to register whether any additional thread has ever been spawned ([evermultithreaded]). *)
```

Comment:
Actually, using `MustBeSingleThreaded {since_start = true}` was our initial approach. But the answer to a `MustTermProg` query should not depend on the current program location.

See https://github.com/goblint/analyzer/pull/1093#discussion_r1242429988

Reply:
I see. Since this is such a tiny analysis, I think it could even be done within the existing threadflag analysis. It shouldn't incur any extra penalty to always have it, especially if it's never queried.

Reply:
I moved it to `threadflag`.